tensorflow/lite/micro/benchmarks/BUILD

```diff
@@ -29,10 +33,10 @@ cc_library(
 cc_library(
     name = "keyword_scrambled_model_data",
     srcs = [
-        "keyword_scrambled_model_data.cc",
+        "//tensorflow/lite/micro/models:generated_keyword_scrambled_model_cc",
```

Comment:
Note (not actionable): There was a debate when I created the models directory which models should live here vs with their example or benchmark. The decision was to only put models here that are needed in >1 place. Still, I like putting the keyword model here since we'll probably share it with the memory footprint example.